src/test/java/org/openmrs/contrib/qaframework/helper/PatientGenerator.java

```diff
@@ -0,0 +1,63 @@
+package org.openmrs.contrib.qaframework.helper;
+
+public class PatientGenerator {
+
+	private static final String[] PATIENT_GIVEN_NAMES = {"Alexandre", "Achint",
```

Comment:
There seems to be some duplication here with the `TestData` class. We should think about reducing that duplication...

Reply:
We need this for rest client

Reply:
Sure, but surely this code can be put into the `TestData` and then we only need one array for the MONTHS, DAYS, and YEARS?

Reply:
ohh sounds good thnks ,Let me refactor that out